packages/dynamite/dynamite_runtime/lib/src/http_client.dart

```diff
@@ -116,9 +116,11 @@ class DynamiteClient {
     final HttpClient? httpClient,
     this.cookieJar,
     this.authentications = const [],
-  }) : httpClient = (httpClient ?? HttpClient())..userAgent = userAgent;
+  })  : httpClient = (httpClient ?? HttpClient())..userAgent = userAgent,
+        assert(!baseURL.path.endsWith('/'), "The baseURL should not end with a '/'"),
+        assert(baseURL.queryParameters.isEmpty, 'The baseURL can not contain query parameters.');
```

Comment:
Is there any reason why we should not allow it? I don't think it could break anything and maybe this is useful somewhere else? :woman_shrugging: